ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -413,6 +427,23 @@ def pytest_configure(config):
                 gather_version_info_for_report(config)
     # switch the configuration context back to the default cluster
     ocsci_config.switch_default_cluster_ctx()
+    pytest_logs_dir = ocsci_log_path()
+    create_directory_path(ocsci_config.RUN["log_dir"])
+    create_directory_path(pytest_logs_dir)
+    config.option.logger_logsdir = os.path.join(pytest_logs_dir, "pytest_logger")
+    config.option.log_file = os.path.join(pytest_logs_dir, "test_info_level.log")
+    logs_url = ocsci_config.RUN.get("logs_url")
+    for i in range(ocsci_config.nclusters):
+        ocsci_config.switch_ctx(i)
+        if logs_url:
+            ocsci_config.RUN["info_logs_url"] = urljoin(
+                logs_url, os.path.basename(pytest_logs_dir)
+            )
+        ocsci_config.REPORTING["log_path_message"] = (
+            f"Logs from run-ci execution (RUN ID: {ocsci_config.RUN['run_id']})"
+            f" will be stored in: {ocsci_config.RUN.get('info_logs_url') or pytest_logs_dir}\n"
+        )
+    print(ocsci_config.REPORTING["log_path_message"])
```

Comment:
This is intentionally printed with the `print` command, not logged through `logger`, right?

Reply:
Yes, this is intentionally to print this out like the pytest is printing out the summary at the end of execution, so this needs to be just printed out without any logging information